src/main/resources/db/migration/V1_6__update_view_mv_list_of_values.sql

```diff
@@ -0,0 +1,80 @@
+DROP MATERIALIZED VIEW mv_list_of_values;
```

Comment:
Any reason for dropping and then creating the view instead of updating the view?

Reply:
good question @SujeethKumarAri  i tried few ways of using update or altering view to accommodate new tables but unfortunately couldn't find  to my knowledge ....any valuable solutions will be appreciated...

Reply:
Sorry, that's a grey area for me. Can you post this in our team channel and see if someone has idea about it?

Reply:
Great Review @SujeethKumarAri  removed the replace view function which i written again ...[placed or not no problem] better to remove

Reply:
thanks for doing the changes @sahitya-desireddy